gcp/basic-gke-cluster/README.md

```diff
@@ -47,51 +47,51 @@ To run the YAML that this blueprint generates, you need:
 
 ## Information required
 
-This blueprint requires:
+This blueprint requires a:
 
-* GCP project ID (see section below for instructions)
-* A GCP region
+* GCP project ID
+* GCP region
 
-### Creating Google Cloud project
+### Create a Google Cloud project
 
 #### Authenticate to gcloud
-Before configuring gcloud CLI you can check available Zones and Regions nearest to your location
+Before configuring gcloud CLI you can check for available Zones and Regions nearest to your location:
 
 ```sh
 gcloud compute regions list
 
 gcloud compute zones list
 ```
 
-Follow gcloud init and select default Zone Ex. europe-west1, make sure you use the same zone as the one selected during blueprint execution.
+Follow `gcloud init` and select default Zone (for example, `europe-west1`). Make sure you use the same zone as the one selected during blueprint execution.
```

Comment:
I don't think "follow" is the right word here. You're executing the `gcloud init` command.

select **a** default zone

I would make the last sentence active voice: _Make sure you use the same zone as the one that you select when you answer the blueprint's questions._

I don't think the end user will necessarily know what "blueprint execution" means; they might not know whether it's "the time when I'm answering the blueprint questions" or "the time when XLR/XLD are running the stuff that the blueprint generated".

Reply:
Done.